buildMirge.sh

```diff
@@ -53,10 +53,19 @@ fi
 
 if [ -z ${CONDA_PATH+x} ]; then
   echo "CONDA_PATH unset, installing new conda with emirge"
-  echo "./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production"
-  ./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production
+  echo "./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy"
+  ./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy
 else
   echo "Using existing Conda installation, ${CONDA_PATH}"
-  echo "./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production"
-  ./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production
+  echo "./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy"
+  ./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy
 fi
+cd mirgecom
+git pull
+git checkout y1-production
+git checkout parallel-lazy
+git checkout -b mrgy1
+git config --global user.email "ci@github.com"
+git config --global user.name "CI Runner"
+git merge y1-production
+cd ../
```

Comment:
I'm not wild about this, seems easily broken as the branches evolve. If we want to test lazy through CI while development is on-going, I think a separate job that does this would be best, instead of wedging it into the build script.

Reply:
This and the other branch stuff above is not intended to merge - only to demonstrate what to do to make lazy runs work with the nozzle.